app/src/main/res/values/strings.xml

```diff
@@ -144,4 +144,13 @@
     <string name="resend">Resend</string>
     <string name="shutdown">Shutdown</string>
     <string name="reboot">Reboot</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="show_intro">Show Introduction</string>
+    <string name="intro_welcome_title">Welcome to Meshtastic</string>
+    <string name="intro_meshtastic_desc">Meshtastic is an open-source, off-grid, encrypted communication platform. The meshtastic radios form a mesh network and communicate using the LoRa protocol to send text messages</string>
+    <string name="intro_get_started">...Let\'s get started!</string>
+    <string name="intro_started_text">Connect your meshtastic device by using either bluetooth or serial. \n\nYou may need to wake your node if they are battery powered. \n\nYou can see which devices are compatible at www.meshtastic.org/docs/hardware</string>
+    <string name="intro_ecryption_title">"Setting up encryption"</string>
+    <string name="intro_encryption_text">To enable encryption, go to the channel tab and change the channel name. This will set a random key and enable AES256 encryption. \n\nTo communicate with other devices they will need to scan your QR code or follow the shared link to configure the channel settings.</string>
```

Comment:
the default channel has a public AES256 key, but encryption is enabled. maybe "to create your own channel and encryption key" ...

Reply:
Have reworded it to reflect your suggestions. Thanks :)